neuroscience/planet.neuroscience.html.erb

```diff
@@ -0,0 +1,111 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <head>
+    <title><%= site.title %></title>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
+    <!-- Google analytics -->
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-60261100-2', 'auto');
+      ga('send', 'pageview');
+
+    </script>
+  </head>
+
+  <body>
+    <nav class="navbar navbar-default navbar-static-top">
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="#"><%= site.title %></a>
+        </div>
+          <ul class="nav navbar-nav navbar-right">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">More Neuroscience <span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a href="http://neuroblog.fedoraproject.org/planet-neuroscientists/">Planet Neuroscientists</a></li>
+                <li><a href="http://home.earthlink.net/~perlewitz/index.html">Computational Neuroscience on the web</a></li>
+              </ul>
+            </li>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Options <span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a href="https://github.com/neurofedora/planet-neuroscience/issues/new">Suggest a new feed</a></li>
+                <li><a href="https://github.com/Aniket-Pradhan/planet-neuroscientists">View Planet source</a></li>
+                <li role="separator" class="divider"></li>
+                <li><a href="https://github.com/Aniket-Pradhan/planet-neuroscientists">View Pluto source</a></li>
+              </ul>
+            </li>
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+    <div class="container">
+      <div class="page-header">
+        <h1><%= site.title %></h1>
+        <p>
+        last updated by <a href="">Pluto</a>
+        <% if site.fetched %>
+          on <%= site.fetched %> 
+        <% end %>
+        on behalf of Ankur Sinha
```

Comment:
@Aniket-Pradhan - based on translation guide [here](https://feedreader.github.io/#create-your-own-templates---template-reference), it seems that the author name as exists in `venus` isn't supported. I hardcoded the name for now, but maybe in the future we could contribute to `pluto` to add this? Thoughts?